### DISCUSSION TITLE
Laravel jetstream installation

### DISCUSSION THREAD

**Discussion by jonphipps** [Laravel-Backpack/community-forum#880] [Q&A (Help)] [answered]
If I start with a jetstream/livewire teams install, what will I need to do to get backpack working with it?

I ask because I have a strong use case for the jetstream user authentication, 2FA, social logins, and teams. It looks like spatie/permissions can be integrated too.

Thanks in advance.

**Comment by jcastroa87** [comment, marked as answer]
Hello @jonphipps 

You can try to install Backpack as normal; if something fails, give it a try with manual installation.

Here are some users who found early a few problems on installation and how they fix:

https://github.com/Laravel-Backpack/community-forum/issues/692
https://github.com/Laravel-Backpack/community-forum/issues/597
https://github.com/Laravel-Backpack/community-forum/discussions/688

We didn't give support for Jetstream, but if you have any issues, come back and ask so we can try to help you.

Cheers.
